lib/controller.rb

```diff
@@ -1,6 +1,11 @@
 class Controller
+
   def initialize(game)
     @game = game
   end
 
+  def play_move
```

Comment:
This method calls two consecutive methods on game. Which suggests to me that this behaviour might belong in game.

Reply:
Thanks, Matt. I wasn't sure where this method should be placed. I'll move this into Game